goutubedl.go

```diff
@@ -211,6 +211,8 @@ type Options struct {
 	DebugLog          Printer
 	StderrFn          func(cmd *exec.Cmd) io.Writer // if not nil, function to get Writer for stderr
 	HTTPClient        *http.Client                  // Client for download thumbnail and subtitles (nil use http.DefaultClient)
+	MergeOutputFormat string                        // --merge-output-format
+	SortingFormat     string                        // --format-sort
```

Comment:
Didn't even know how --format-sort worked, interesting. Is `--format-sort-force` interesting also? could be added later also